app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :name, presence: true, length: {maximum: Settings.user.name_max}
+  validates :email, presence: true, length: {minium: Settings.user.mail_min,
+                                             maximum: Settings.user.mail_max},
+    format: {with: Settings.user.mail_regex},
+    uniqueness: {case_sensitive: false}
```

Comment:
uniqueness: {case_sensitive: false} có tác dụng gì em

Reply:
theo em case_sensitive: false để đảm bảo DB không lưu trường này vào nếu các phiên bản in hoa, in thường của nó đã tồn tại